content/en/docs/Reference/artifacts.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Artifact Naming Convention"
+linkTitle: "Artifact Naming Convention"
+weight: 1
+date: 2023-07-19
+description: >
+  Detailed information about how we name our artifacts including repositories.
+---
+
+# Artifacts
+
+Artifacts are named using the following convention:
+
+```
+kairos-<support>-<type>-<flavor>-<arch>-<device>-<version>[-<non-image-type>].<extension>
+```
+
+Where:
+
+- `<support>`: `official` or `community`
+- `<type>`: `core` or `standard`
+- `<flavor>`: Underlying Linux distribution e.g. `ubuntu`, `debian`, `fedora`, `alpine`, etc
+- `<arch>`: `x86_64`, `arm64`, `armhf`, `aarch64`, etc
+- `<device>`: either the specific device name, e.g. `rpi64` or `generic` for generic images
+- `<version>`: the version of Kairos e.g. `1.0.0` can also include the K3S version e.g. `1.0.0-k3s1` in case of Kairos Standard
+- `<non-image-type>`: only used for artifacts that are not images e.g. `trivy` for Trivy scans
+
+## Examples
+
+- `kairos-official-core-ubuntu-x86_64-generic-1.0.0.iso`
+- `kairos-official-standard-ubuntu-arm64-rpi64-1.0.0-k3s1.img`
+- `kairos-community-core-alpine-aarch64-generic-1.0.0-trivy.sarif`
```

Comment:
The mix of arm64 and aarch64 is very confusing, aren't they the same?

Reply:
yeah good point, will fix ... also I see we have in some places x86_64 but translate to amd64, do you know what is prefered?

Reply:
Depends on the day and location lol.

Kernel reports x86 and aarch64, golang says x86 and amd64 and goreleaser does a mix I think

We just need one and use it over everywhere

Reply:
Ok, went with the golang version, just because it's documented in one place but also left a notice at the beginning of the document so users can be aware of this